RadixWallet/Features/AssetTransferFeature/Components/TransferAccountList/TransferAccountList+View.swift

```diff
@@ -89,12 +89,12 @@ private extension View {
 		}
 	}
 
-	private func addAsset(with destinationStore: PresentationStoreOf<TransferAccountList.Destination>) -> some View {
+	private func addAsset(with destinationStore: PresentationStoreOf<TransferAccountList.Destination>, store: StoreOf<TransferAccountList>) -> some View {
 		sheet(store: destinationStore.scope(state: \.state.addAsset, action: \.addAsset)) { assetsStore in
 			AssetsView.View(store: assetsStore)
 				.radixToolbar(title: L10n.AssetTransfer.AddAssets.navigationTitle)
 				.withNavigationBar {
-					assetsStore.send(.view(.closeButtonTapped))
```

Comment:
We were sending an action to `AssetsView` (which doesn't have a close button but had a `closeButtonTapped` action), and such view would internally call the delegate that was handled by this `TransferAccountList` reducer.. better just send the action directly to the `TransferAccountList`.